src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/HttpClientHandler.Browser.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Security;
+using System.Runtime.Versioning;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using HttpHandlerType = System.Net.Http.BrowserHttpHandler;
+
+namespace System.Net.Http
+{
+    public partial class HttpClientHandler
+    {
+        private readonly HttpHandlerType _underlyingHandler;
+
+        public HttpClientHandler()
+        {
+            _underlyingHandler = new HttpHandlerType();
+            if (DiagnosticsHandler.IsGloballyEnabled())
+            {
+                _diagnosticsHandler = new DiagnosticsHandler(_underlyingHandler);
+            }
+            ClientCertificateOptions = ClientCertificateOption.Manual;
+        }
+
+        public virtual bool SupportsAutomaticDecompression => HttpHandlerType.SupportsAutomaticDecompression;
+        public virtual bool SupportsProxy => HttpHandlerType.SupportsProxy;
+        public virtual bool SupportsRedirectConfiguration => HttpHandlerType.SupportsRedirectConfiguration;
+
+        [UnsupportedOSPlatform("browser")]
+        public bool UseCookies
+        {
+            get => throw new PlatformNotSupportedException();
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
Can't the setter also use `=>` ?

Reply:
no, it is not returning a value

Reply:
Well, it works for me: https://dotnetfiddle.net/wiEZZz

This compiles and works:
```C#
using System;
					
public class Program
{
	public int X
	{
		get => throw new Exception("Getter");
		set => throw new Exception("Setter");
	}
	public static void Main()
	{
		new Program().X = 5;
	}
}
```

Reply:
Right, i mistaken